lib/MusicBrainz/Server/Form/User/Register.pm

```diff
@@ -10,7 +11,8 @@ has '+name' => ( default => 'register' );
 has_field 'username' => (
     type      => 'Text',
     required  => 1,
-    maxlength => 64
+    maxlength => 64,
+    validate_method => \&validate_username
```

Comment:
Nothing else in this file has the trailing comma, so I didn't add it because then I'd have to add it everywhere and then it wouldn't belong in this PR anymore I guess...